src/index.jsx

```diff
@@ -1,4 +1,14 @@
 const axios = require('axios');
+<<<<<<< HEAD
```

Comment:
Does this need to be removed prior to merging?

Reply:
Yep, just missed that before I did the PR.